Signal/src/ViewControllers/ConversationView/ConversationInputToolbar.m

```diff
@@ -127,6 +128,8 @@ - (void)createContents
     _composeContainer = [UIView containerView];
     _contentStackView = [[UIStackView alloc] initWithArrangedSubviews:@[ _composeContainer ]];
     _contentStackView.axis = UILayoutConstraintAxisVertical;
+    _contentStackView.layoutMargins = UIEdgeInsetsZero;
```

Comment:
I see now why we're doing this, but by itself this is kind of inexplicable, since enabling layoutMarginsRelativeArrangement with layoutMargins of 0 is a noop. See my comment below...